code/modules/mob/living/carbon/xenomorph/Castes/Sentinel.dm

```diff
@@ -154,12 +154,78 @@
 	upgrade = 0
 	speed = -0.8
 	pull_speed = -2
+	var/last_neurotoxin_sting = null
 	actions = list(
 		/datum/action/xeno_action/xeno_resting,
 		/datum/action/xeno_action/regurgitate,
 		/datum/action/xeno_action/activable/corrosive_acid/drone,
 		/datum/action/xeno_action/activable/xeno_spit,
+		/datum/action/xeno_action/activable/neurotox_sting,
 		)
 	inherent_verbs = list(
 		/mob/living/carbon/Xenomorph/proc/vent_crawl,
 		)
+
+
+/mob/living/carbon/Xenomorph/Sentinel/proc/neurotoxin_sting(var/mob/living/H)
+
+	if(!check_state()) return
+
+	if(world.time < last_neurotoxin_sting + NEUROTOXIN_STING_COOLDOWN) //Sure, let's use this.
+		to_chat(src, "<span class='xenowarning'>You are not ready to use the sting again. It will be ready in [(last_neurotoxin_sting + NEUROTOXIN_STING_COOLDOWN - world.time) * 0.1] seconds.</span>")
```

Comment:
```suggestion
		to_chat(src, "<span class='xenowarning'>You are not ready to use the sting again. It will be ready in [(last_neurotoxin_sting + NEUROTOXIN_STING_COOLDOWN - world.time) SECONDS] seconds.</span>")
```